projects/packages/jetpack-mu-wpcom/src/features/wpcom-profile-settings/profile-settings-link-to-wpcom.php

```diff
@@ -51,4 +55,4 @@ function wpcom_profile_settings_add_links_to_wpcom() {
 		)
 	);
 }
-add_action( 'load-profile.php', 'wpcom_profile_settings_add_links_to_wpcom' );
+add_action( 'profile_personal_options', 'wpcom_profile_settings_add_links_to_wpcom' );
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blue.svg) Is there a specific reason you prefer `profile_personal_options`?

Reply:
It is a more specific hook. I'd prefer using the most specific hook that still does the job, for safety reason. Although, in this specific case, it does not really matter as that hook is only called on `profile.php` 😆